openlcsd/flow/tasks.py

```diff
@@ -963,6 +963,18 @@ def get_module_components_from_corgi(context, engine):
     engine.logger.info("Finished getting module components data.")
 
 
+def get_module_components_from_brew(context, engine):
+    """
+    Get module components from brew
+    """
+    config = context.get('config')
+    module_nvr = context.get('package_nvr')
+    engine.logger.info("Start to get module components data...")
+    koji_connector = KojiConnector(config)
+    context['components'] = koji_connector.get_module_components(module_nvr)
+    engine.logger.info("Finished getting module components data.")
+
+
 def fork_specified_type_imports(
         context, engine, nvr_list, src_dir, comp_type):
```

Comment:
set default value for `src_dir`

Reply:
Updated.

Reply:
If we change the parameter location, we should update all the functions that use this function

Reply:
Updated all the call of fork_specified_type_imports. Thanks.